lib/utils.js

```diff
@@ -317,6 +318,32 @@ function isProhibited(key, prohibitedKeys) {
   return false;
 }
 
+/**
+ * Accept an operator key and return whether it is used for a regular expression query or not
+ * @param {string} operator
+ * @returns {boolean}
+ */
+function isRegExpOperator(operator) {
```

Comment:
what about other operators?

https://loopback.io/doc/en/lb4/apidocs.repository.operators.html

![image](https://user-images.githubusercontent.com/6864736/66843837-52a27c00-ef3b-11e9-8126-7d4edc7ac7ce.png)

Reply:
@emonddr This is for regexp related operators only.

Reply:
@emonddr as @raymondfeng said this targets the crash caused by Invalid regular expression operators